pandas/core/frame.py

```diff
@@ -6664,6 +6668,19 @@ def corr(self, method='pearson', min_periods=1):
         Returns
         -------
         y : DataFrame
+
+        Examples
+        --------
+        >>> histogram_intersection = lambda a, b: np.minimum(
+        ...    np.true_divide(a, a.sum()), np.true_divide(b, b.sum())
+        ... ).sum()
+
+        >>> df = pd.DataFrame([(1, 2), (0, 3), (2, 0), (1, 1)],
+        ...                   columns=['dogs', 'cats'])
+        >>> df.corr(method = histogram_intersection)
```

Comment:
This is failing our doctests. Is there an issue with the output?

Also, pep8: no spaces around the `=`